### DISCUSSION TITLE
Editing is not possible

### DISCUSSION THREAD

**Discussion by aaro-n** [requarks/wiki#5806] [Error / Bug Report] [answered]
system message:
Version number: Wiki.js 2.5.290
Deployment method: Docker
Database: PostgreSQL
Problem Description:
I recently upgraded to the latest version, but I didn't notice if I could edit it. Today I opened "Network" in Chrome, and then clicked "Edit" on the wiki. The browser was blank`/_assets/css/editor.5bcab9560469306e3967.css` 404
Other examples:
https://docs.requarks.io/_assets/css/editor.5bcab9560469306e3967.css

**Comment by NGPixel** [comment, marked as answer]
Clear your browser cache. If you use a reverse-proxy in front of your wiki, make sure to clear its cache as well.

**Comment by aaro-n** [comment]
Refreshing the CDN did the trick, I use aws cloudfront.
